pkg/apiclient/event/event.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+option go_package = "github.com/argoproj/argo/pkg/apiclient/event";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto";
+import "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1/generated.proto";
+
+package event;
+
+message EventRequest {
+    // The namespace for the event. This can be empty - i.e. the client has cluster scoped permissions and want this dispatched to all workflows in
+    // all namespaces, or if set - only to workflows in the specified namespace
+    string namespace = 1;
```

Comment:
Can you clarify here? With this statement, are you saying that we support a use case where a single event can "broadcast" to all WorkflowEventBindings in all namespaces in the cluster, and trigger different workflow templates in their corresponding namespace?

Reply:
The workflow is always created in the namespace of the event binding. `namespace=""` would  trigger workflows event bindings in all namespaces.